### PR TITLE
430 - Phase Out Open Call for Contributions

### LINKED ISSUE #430: Phase Out Open Call for Contributions
### Context

We are no longer open for contributions. We want to remove references of the open call to contributions from the site.

### Solution or next step

- Remove contribution banner on home page
- Update `/contribute` page
  - add card: https://github.com/AlexsLemonade/ScPCA-admin/issues/621#issuecomment-1707065958
  - remove `Download Guidelines as PDF` button
  - Update config `contribute_hsform` to: https://share.hsforms.com/1Fs_Z94OzS7abnh5qEieOTA336z0
  - delete the guidelines pdf in `/src/public`

Note: Markdown version of the guidelines will be updated in it's host repo and merged in a separate PR

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number

Closes #430

## Purpose/Implementation Notes

Based on the requirement, removed the contribute banner and display the contribute page card in the contribute page.

@dvenprasad, please review the latest UI [here](https://scpca-portal-l3yun0vqa-ccdl.vercel.app).

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by davidsmejia — CHANGES_REQUESTED** [review]
This looks good, I don't think we need to worry about rendering this specific card on other pages by design. Also there was a layout thing with the padding at my browser width, which moving it into the page would resolve.

This would be LGTM to me after:
- Renaming `ContributePageCard` to `WarningCard` and pass in the props
- Move the `ContributePageCard` from the layout to where the button was on the `contribute` page.

**Comment by nozomione** [comment]
> I don't think we need to worry about rendering this specific card on other pages by design. Also there was a layout thing with the padding at my browser width, which moving it into the page would resolve.
>
>     * Renaming `ContributePageCard` to `WarningCard` and pass in the props

Is this warning card design specific to the contribute page? Is there any plans to create warning cards at the page level with a similar style 🎨 ? If the latter is the case, then It makes sense. 

For the red warning text, currently our shared `WarningText` component is used.


>     * Move the `ContributePageCard` from the layout to where the button was on the `contribute` page.

Please see the screenshot below (the first one is in the `Layout` component and the second one is in the contribute page component):

*I added the green text in the middle 😄  

Screenshot 2023-09-13 at 6 16 19 PM

Due to the way the site structure is currently set up, the contribute page width is fixed (not full width against a browser) and for this reason, the yellow and blue band cuts off. To move this card to the contribute page component from `Layout`, We'll need to add a new JS condition in `Layout` to make this page width to be full, and then adjust this page styles (e.g., by introducing additional `div` to make the content width to be fixed). So it's much simpler to keep the current implementation.  

Also, I made improvements to the site structure in this PR https://github.com/AlexsLemonade/scpca-portal/pull/405, which simplifies and makes the site layout more flexible (also addresses the padding condition issues for pages) and remove most of the JS conditions (we previously discussed cleaning them up). Could we review it so that it may be merged?

**Comment by jaclyn-taroni** [comment]
I wanted to add a few notes. First, Deepa and I discussed this before she went on PTO, and she said she trusted the rest of the team to make decisions to get something posted as soon as possible. That's to say, we should not wait for Deepa's review of this PR. We may need to tweak it once she returns, but that's okay.

Because we would like the changes in #430 to go out as soon as possible, can we find a strategy for getting this merged that doesn't rely on the review, approval, and merging of #405, please?

**Comment by nozomione** [comment]
> I wanted to add a few notes. First, Deepa and I discussed this before she went on PTO, and she said she trusted the rest of the team to make decisions to get something posted as soon as possible. That's to say, we should not wait for Deepa's review of this PR. We may need to tweak it once she returns, but that's okay.
> 
> Because we would like the changes in #430 to go out as soon as possible, can we find a strategy for getting this merged that doesn't rely on the review, approval, and merging of #405, please?

@jaclyn-taroni  I see. In this case, I removed Deepa as a reviewer. 

@davidsmejia If the contributePage card must be in the contribute page, then I'll need to revise the page layout. Can we merge this PR as is for now and we'll make the site structure improvement (#405) after that? Please let me know.

**Comment by davidsmejia** [comment]
As per our meeting today lets:


- Rename ContributePageCard to WarningCard and pass in the props
- Move the ContributePageCard from the layout to where the button was on the contribute page.
- Add some drop shadow to the card and add some padding to the text contents

**Comment by nozomione** [comment]
@davidsmejia Updated, please review the latest UI [here](https://scpca-portal-p62jt24nk-ccdl.vercel.app/contribute), thank you.

File: client/src/components/Layout.js
Comment (davidsmejia) [inline client/src/components/Layout.js:57]: I think removing contribute here would make the page more consistent with the designs.
We typically only have the pad removed on pages where the hero should bump up against the nav bar.

**Review by davidsmejia — APPROVED** [review]
LGTM

Left a comment based on my interpretation of the designs but we can resolve it when depa is back

**Pull request merged**
